dipy/__init__.py

```diff
@@ -35,12 +37,39 @@
 """
 import sys
 
-from .info import __version__
+from dipy.version import version as __version__
 from .testing import setup_test
 
 # Plumb in version etc info stuff
 from .pkg_info import get_pkg_info as _get_pkg_info
+
+
 def get_info():
     from os.path import dirname
     return _get_pkg_info(dirname(__file__))
+
+
 del sys
+
+submodules = [
```

Comment:
Would it be possible to automate this with `setuptools.find_packages`?

Reply:
I suppose yes and open to any suggestions